modules/setting/cors.go

```diff
@@ -28,7 +28,7 @@ func newCORSService() {
 
 	CORSConfig = cors.Options{
 		Scheme:           sec.Key("SCHEME").String(),
-		AllowDomain:      sec.Key("ALLOW_DOMAIN").String(),
+		AllowDomain:      sec.Key("ALLOW_DOMAIN").Strings(","),
```

Comment:
This looks unrelated

Reply:
This was changed in the gitea fork. https://gitea.com/macaron/cors/commit/609039acef4c33132ea87a69cd69ba826127eff3

Build fails without it.

Reply:
Oh, I see, it's because Macaron now expects a slice.

Reply:
Yes this is related to the issue #7840 that could be closed after this PR.

Reply:
@sapk No because point 3. is not yet solved